libs/hdf-converters/src/sonarqube-mapper.ts

```diff
@@ -192,8 +210,13 @@ export class SonarQubeResults {
 }
 
 function createSonarqubeMappings(
-  projectName: string
+  projectName: string,
+  branchName?: string,
+  pullRequestID?: string
 ): MappedTransform<ExecJSON.Execution, ILookupPath> {
+  const scanDescriptionModifier = (
```

Comment:
getting ahead of the linter, it usually prefers single quotes and paired indentation levels for things like braces and parentheses